test/common/stats/allocator_impl_test.cc

```diff
@@ -149,7 +172,7 @@ TEST_F(AllocatorImplTest, ForEachCounter) {
   size_t num_iterations = 0;
   alloc_.forEachCounter([&num_counters](std::size_t size) { num_counters = size; },
                         [&num_iterations, &stat_names](Stats::Counter& counter) {
-                          EXPECT_EQ(stat_names.count(counter.statName()), 1);
+                          EXPECT_NE(stat_names.find(counter.statName()), stat_names.end());
```

Comment:
What you have is fine, but I actually think previous state of this code, with the calls to `count` with EXPECT_EQ to 1 would produce a more useful error message if they fail. stat_names.end() will not have an especially readable printout.

Up to you.

Reply:
Fixed.